### PR TITLE
Add slicing support to the vector binding

### PR DESCRIPTION
**Pull request by AndrejMitrovic**

### CODE REVIEW

File: source/scpd/Cpp.d [hunk 1]
```diff
@@ -154,6 +154,18 @@ extern(C++, (StdNS!())) struct vector (T, Alloc = allocator!T)
             return ConstIterator(this._start, this);
         }
 
+        public T[] opSlice ()
```
Comment (Geod24) [inline source/scpd/Cpp.d]: `inout pure nothrow @nogc`

**Comment by AndrejMitrovic** [comment]
Added keyword soup.

File: source/scpd/Cpp.d [hunk 2]
```diff
@@ -154,6 +154,19 @@ extern(C++, (StdNS!())) struct vector (T, Alloc = allocator!T)
             return ConstIterator(this._start, this);
         }
 
+        public inout(T[]) opSlice () inout pure nothrow @nogc @trusted
```
Comment (Geod24) [inline source/scpd/Cpp.d]:
```suggestion
        public inout(T[]) opSlice () inout pure nothrow @nogc @safe
```

**Review by Geod24 — APPROVED** [review]
One tiny fix oLGTM

**Review by Geod24 — APPROVED** [review]

**Pull request merged**